low-level/sspl_ll_d

```diff
@@ -544,42 +565,64 @@ def _sensors_actuators_factory(sspl_threaded_modules, msgQlist, operating_system
     return (sspl_threaded_modules, msgQlist)
 
 
-def _run_thread_capture_errors(curr_module, sspl_threaded_modules, msgQlist, conf_reader, product, resume):
-    """Run the given thread and log any errors that happen on it.
-    Will stop all sspl_threaded_modules if one of them fails."""
-    try:
-        # Suspend module threads
-        if resume == False:
-            curr_module.suspend()
-        # Each module is passed a reference list to message queues so it can transmit
-        # internal messages to other modules as desired
-        logger.info("Starting: %s" % curr_module.name())
-        curr_module.start_thread(conf_reader, msgQlist, product)
-
-    except ThreadException as ex:
-        #TODO: Restart thread instead of shutdown
-        logger.error("%s, error is unrecoverable , shutting down %s" %
-            (ex, curr_module.name()))
-        curr_module.shutdown()
-    except Exception as ex:
-        # Populate an actuator response message and transmit back to HAlon that we have a fatal error
-        # error_msg = "SSPL-LL encountered an error, terminating service Error, restarting daemon:{}" \
-        #             .format(str(logger.exception(ex)))
-        # jsonMsg   = ThreadControllerMsg(curr_module.name(), error_msg).getJson()
-        # curr_module._write_internal_msgQ(EgressProcessor.name(), jsonMsg)
-
-        # Shut down any running threads, error is non-recoverable
-        # shutdown_handler()
-
-        # Pause for threads to cleanup and kill main process
-        # logger.info("Killing main process in 10 seconds to allow thread cleanup time")
-        # time.sleep(10)
-        # logger.info("Systemd will restart SSPL-LL momentarily")
-        # os.kill(os.getpid(), signal.SIGKILL)
-
-        #TODO: Restart thread instead of shutdown
-        logger.error(f"{curr_module.name()} has encountered an error {ex}, error is unrecoverable , shutting down {curr_module.name()}")
-        curr_module.shutdown()
+def _run_thread_capture_errors(module, msgQlist, conf_reader, product, resume):
```

Comment:
update name 
_run_thread_capture_errors() -> _run_thread()

Reply:
This method is now available in thread_controller.py.
Renamed.

Reply:
Done.